packages/e2e/cypress/e2e/app/dates.cy.ts

```diff
@@ -390,7 +390,7 @@ describe('Date tests', () => {
         cy.get('.tabler-icon-x').click({ multiple: true });
     });
 
-    it('Should keep value when changing date operator', () => {
+    it.only('Should keep value when changing date operator', () => {
```

Comment:
only

Reply:
good catch, I always forget about reverting those...

Reply:
The other e2e test is fixed here https://github.com/lightdash/lightdash/pull/10835 feel free to review that one too if you have time